composer.json

```diff
@@ -19,7 +19,7 @@
 
     "require": {
         "php":                      ">=5.3.9",
-        "symfony/framework-bundle": "~2.1",
+        "symfony/framework-bundle": "~2.3|~3.0",
```

Comment:
`2.1` and `2.2` branches are not supported [anymore by symfony community](http://symfony.com/roadmap#checker), so I am not sure that we should and *why* we should.